src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -2589,432 +2589,434 @@ <h2>{{- getValue "notes-page-count"}}. Accounting policies </h2>
           {{end}}
           <!-- ACCOUNTING POLICIES END -->
 
-        {{ if $tangibleAssetsNote := $additionalNotes.tangible_assets }}
-            {{incValue "notes-page-count" }}
-            {{setValue "tangible-notes-id" (getValue "notes-page-count" ) }}
-            <div class="page">
-                <div class="header">
-                    <h1 class="text--center">
+        {{end}}
+        <!-- NOTES END -->
+
+        <!-- BALANCE SHEET NOTES START -->
+        {{if $balanceSheetNotes := .small_full_accounts.balance_sheet_notes}}
```

Comment:
Could we add similar comments for tangible note start and finish? Not essential, just makes for easier navigation in the template